### PR TITLE
[MRG] Make setup.py idiomatic

### PR DESCRIPTION
**Pull request by smu095**
This PR addresses issue #13.

Notes:
- Added @tommyod , @smu095 and @glemvik to list of maintainers.
- Added project url: https://github.com/tommyod/treedoc
- Created a gmail-account, treedoc.dev@gmail.com, for maintenance issues.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
-    description="Minimalistic Python documentation in a tree structure.",
-    long_description="Minimalistic Python documentation in a tree structure.",
+    description="Minimalistic Python documentation for dendrophiles.",
+    long_description=long_description,
+    url="https://github.com/tommyod/treedoc",
     author="tommyod",
     author_email="tod001@uib.no",
+    maintainer="tommyod, smu095, glemvik",
+    maintainer_email="treedoc.dev@gmail.com",
     license="MIT",
```
Comment (tommyod) [inline setup.py]: Let's decide on a license and upload a LICENSE file too. See [choosealicense.com](https://choosealicense.com/licenses/). MIT is very permissive, and I see no reason not to use it. Thoughts?

File: setup.py [hunk 2]
```diff
@@ -1,12 +1,21 @@
+from os import path
 from setuptools import find_packages, setup
 
+# Get the long description from README.md
+PATH = path.abspath(path.dirname(__file__))
+with open(path.join(PATH, "README.md"), encoding="utf-8") as f:
```
Comment (tommyod) [inline setup.py]: Nitpicking, but no single-letter variable names. Let's call it `file`.

File: setup.py [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
+    long_description = f.read()
+
 setup(
     name="treedoc",
     version="0.2.0",
-    description="Minimalistic Python documentation in a tree structure.",
-    long_description="Minimalistic Python documentation in a tree structure.",
+    description="Minimalistic Python documentation for dendrophiles.",
```
Comment (tommyod) [inline setup.py]: `grep` for the old tagline and replace it everywhere in the coce.

File: setup.py (hunk 1 above)
Comment (smu095) [reply setup.py]: No strong feelings, as far as I can tell the MIT license is quite popular. I suggest we go for it. I've made a `LICENCE.md`, will upload next commit.

Comment (tommyod) [reply setup.py]: Great!

**Comment by smu095** [comment]
I'm getting an error in Travis related to `mypy`. Any ideas?

**Pull request merged**